rlog/state-separation.mdx

```diff
@@ -0,0 +1,1255 @@
+---
+title: 'Nescience - A user-centric state separation architecture'
+date: 2024-08-07 12:00:00
+authors: moudy
```

Comment:
Consider capitalising "Moudy"

Reply:
in the file Authors it is defined as "moudy" and shows as "Moudy"